paddle/function/nnpack/NNPACKConvOp.cpp

```diff
@@ -196,10 +196,13 @@ class NNPACKConvFunction : public ConvFunctionBase {
         CHECK_EQ(status, nnp_status_success);
       }
     } else {
+      // only supports stride = 1
+      CHECK_EQ(strideH(), 1);
+      CHECK_EQ(strideW(), 1);
+
+      // TODO(hedaoyuan): There has some bug when batchSize > 1 and groups_ > 1.
+      CHECK_EQ(groups_, (size_t)1);
```

Comment:
`(size_t)1` ==> `size_t(1)` or `static_cast<size_t>(1)`, according to Google C++ style https://google.github.io/styleguide/cppguide.html#Casting